vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -622,7 +622,7 @@ supported:
       </ComplexType>
 
       <ComplexType Name="PermissionType">
-        <Property Name="Scheme" Type="Auth.SecurityScheme" Nullable="false">
+        <Property Name="Scheme" Type="Edm.String" Nullable="false">

```

Comment:
If this corresponds to 

Authorization | String | The name of a required authorization scheme
-- | -- | --

Then it should better be named `AuthorizationSchemeName` or just `SchemeName` because it is defined in the Authorization vocabulary..

@mikepizzo instead of relying on names we could add a type definition `SchemeName` of type `String` to the Authorization vocabulary and use this type definition consistently here and within the Authorization vocabulary.

What do you think?

Reply:
I had the same idea of defining a SchemeName type definition to be more self-documenting, although it does add one more type to resolve. We can discuss in our F2F.